src/core/router.go

```diff
@@ -112,6 +112,7 @@ func initRouters() {
 
 	beego.Router("/api/internal/syncregistry", &api.InternalAPI{}, "post:SyncRegistry")
 	beego.Router("/api/internal/renameadmin", &api.InternalAPI{}, "post:RenameAdmin")
+	beego.Router("/api/internal/configurations", &api.ConfigAPI{}, "get:GetInternalConfig")
```

Comment:
Will this be a formal route in future?

Reply:
It is not the final version, introduce /api/internal/configurations is used only for remove dependency and remove admin container.